e2e_test/sink/aggregation.slt

```diff
@@ -0,0 +1,17 @@
+statement ok
```

Comment:
Is there any plan to run this in CI?

Reply:
This test file slipped in there by accident. I used it to fix some unexpected behavior. I see you've already created an issue for it (https://github.com/singularity-data/risingwave/issues/4067) thanks!.